daemon/inertiad/auth/users.go

```diff
@@ -20,9 +20,11 @@ const (
 // userProps are properties associated with user, used
 // for database entries
 type userProps struct {
-	HashedPassword string
-	Admin          bool
-	LoginAttempts  int
+	HashedPassword  string
+	Admin           bool
+	LoginAttempts   int
+	totpKey         string
```

Comment:
Shouldn't this be exported?

Reply:
This is also going to need a totpSalt key as PBKDF2 requires a salt